control/tap-ctl-spawn.c

```diff
@@ -196,5 +232,15 @@ tap_ctl_spawn(void)
 	if (id < 0)
 		EPRINTF("get_id failed, child %d err %d\n", child, errno);
 
+	/* Put the tapdisk in a cgroup slice (best-effort) */
+	if (!slice) {
+#ifdef TAP_CTL_USE_DEFAULT_CGROUP_SLICE
```

Comment:
I guess that, with this one, you want to make cgroup usage a compile-time switch. If it were a run-time option I would be more happy but
this is something that can be added later one. Just let me know if you think that it does not make sense to be
a run-time option.

However, you define it few line above, so I don't see how the \#ifdef code is not to be executed.

Please, clarify your intention.